### DISCUSSION TITLE
EKS cluster by default it creates it with extended support type

### DISCUSSION THREAD

**Discussion by pablofilgueira91** [upbound/provider-aws#1445] [Q&A] [answered]
Hi, I’m creating an EKS cluster using Crossplane and by default it creates it with extended support type, is there a way to select standard?
I am selecting version 1.30 which is allowed with standard support but still by default it enables extended support. https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
![image](https://github.com/user-attachments/assets/bbbb5736-0374-476b-86d4-a31ae79ecf42)

**Comment by mergenci** [comment, marked as answer]
Terraform allows configuration of EKS support type through [`upgrade_policy` field](https://registry.terraform.io/providers/hashicorp/aws/5.61.0/docs/resources/eks_cluster#upgrade_policy). That field was introduced in v5.61.0, therefore it is not available in our Crossplane provider as of now (v1.10.0), which is [based on Terraform provider v5.58.0](https://github.com/crossplane-contrib/provider-upjet-aws/blob/v1.10.0/Makefile#L14). The field will become available the next time we bump Terraform provider dependency.
